test/e2e/feature/feature.go

```diff
@@ -231,8 +232,9 @@ var (
 	// Ingress.networking.k8s.io to be present.
 	Ingress = framework.WithFeature(framework.ValidFeatures.Add("Ingress"))
 
-	// TODO: document the feature (owning SIG, when to use this feature for a test)
-	InPlacePodVerticalScaling = framework.WithFeature(framework.ValidFeatures.Add("InPlacePodVerticalScaling"))
+	// Owner: sig-node
+	// Marks tests that require InPlacePodVerticalScaling to be enabled.
+	InPlacePodVerticalScaling = framework.WithFeatureGate(features.InPlacePodVerticalScaling)
```

Comment:
Delete this and replace the places it is used with the WithFeatureGate bit

Reply:
We only have this file because these "features" have been defined for e2e tests specifically, otherwise we directly reference the featuregates within the tests, using the standard kubefeatures names / import.

I think it's clearer that way, if I see a `e2e/features.Foo` I'm going to assume it's an e2e "feature" instead of a featuregate